### ISSUE TITLE
Cannot build app with `firebase-ios-sdk` v10.24.0

### ISSUE THREAD

**Issue by ski-u** [akaffenberger/firebase-ios-sdk-xcframeworks#77]
Hi, thank you for working for this project! 👋 
I submit an issue because I have a problem with the new version (10.24.0)

## Description
- I can't build our app with `akaffenberger/firebase-ios-sdk-xcframeworks` 10.24.0
  - Errors said that the signatures can't be verified
- If I add `firebase/firebase-ios-sdk` directly as a SPM package, I can build our app 🙆 

<img width="2032" alt="Screenshot 2024-04-17 at 14 43 16" src="https://github.com/akaffenberger/firebase-ios-sdk-xcframeworks/assets/37182704/54f5dfab-1bc8-4070-bf5a-ecab06fd6305">

## Reproduce
- I made a repository that contains minimum projects to reproduce this issue
  - https://github.com/ski-u/firebase-ios-sdk-signature
- We may have the same error when building `iOS Example` project in this repo

## Environment

```
❯ swift --version
swift-driver version: 1.90.11.1 Apple Swift version 5.10 (swiftlang-5.10.0.13 clang-1500.3.9.4)
Target: arm64-apple-macosx14.0
```

```
❯ xcodebuild -version
Xcode 15.3
Build version 15E204a
```

**Comment by mathaeus** [comment]
+1 here

**Comment by formaad** [comment]
+1

**Comment by tareksabry1337** [comment]
+1

**Comment by lkuczborski** [comment]
+1 @akaffenberger Will you be able to take a look? :)

**Comment by MartinPrusa** [comment]
This issue is reported on Firebase Github https://github.com/firebase/firebase-ios-sdk/issues/12800, fix must be done on their side

**Comment by am-play** [comment]
@akaffenberger It looks like google may have fixed this in 10.25.0: https://github.com/firebase/firebase-ios-sdk/issues/12800#issuecomment-2059516188 Could you update to 10.25.0? Thanks!

**Comment by alexfringes** [comment]
@am-play Just FYI, 10.25.0 isn't the current version on the official Firebase iOS repo yet.

**Comment by akaffenberger** [comment]
Hey all, sorry for the delay, updating it now.

**Comment by lkuczborski** [comment]
Hmm... having same issues in 10.25.0 too :(

**Comment by mathaeus** [comment]
Also still experiencing this issue with `10.25.0`.

**Comment by lkuczborski** [comment]
So it appears that these binaries are not signed properly. Signature are required for binary dependencies as described here: https://developer.apple.com/support/third-party-SDK-requirements/

<img width="877" alt="Screenshot 2024-05-06 at 22 30 03" src="https://github.com/akaffenberger/firebase-ios-sdk-xcframeworks/assets/6804537/63f57415-5723-41f5-8762-7411c7af49ab">

**Comment by ski-u** [comment]
@akaffenberger 
We're still having the same issue with `10.25.0` 😢 
Will you be able to take a look?

We would like to update `firebase-ios-sdk` to the latest version since it contains some updates related to Apple Privacy Manifest!

**Comment by daniapopov** [comment]
I have put up a PR that I think addresses the issue

**Comment by Niccari** [comment]
Version 10.29.0 seems to have resolved the issue. Thank you @daniapopov and @akaffenberger !

**Comment by ski-u** [comment]
I also found this issue has been fixed on v10.29.0! Thank you so much! @daniapopov @akaffenberger 👍

**Comment by akaffenberger** [comment]
Great! I'll close this one! Thanks again @daniapopov!
